lib/dry/system/booter.rb

```diff
@@ -54,6 +63,13 @@ def finalize!
         freeze
       end
 
+      # @!method finalized?
+      #   Returns true if the booter has been finalized
+      #
+      #   @return [Boolean]
+      #   @api private
+      alias_method :finalized?, :frozen?
```

Comment:
Added this so we could use it inside `find_component`, since it much more naturally expresses the state of the `Booter` than `frozen?`, given that `finalize!` is the method that we call to finalize the booter.